Maths/ExponentialFunction.js

```diff
@@ -0,0 +1,24 @@
+let powerOfX = 1
+let factorial = 1
```

Comment:
Try not using global variables.

Reply:
I will update them as you have suggested.